scipy/stats/tests/test_distributions.py

```diff
@@ -573,6 +573,21 @@ def test_logsf(self):
         result = stats.hypergeom.logsf(k, M, n, N)
         exspected = -2239.771   # From R
         assert_almost_equal(result, exspected, decimal=3)
+    
+    def test_logcdf(self):
+        # Test logcdf for very large numbers. See issue #8692
+        # Results compare with those from R (v3.3.2):
+        # phyper(k, n, M-n, N, lower.tail=TRUE, log.p=TRUE)
+        # -5273.335
+
+        k = 1
+        M = 1e7
+        n = 1e6
+        N = 5e4
```

Comment:
Just one thought here -- it does indeed look like you are doing a good job of replicating the test format used above with a comparison to R, but it may also be useful to show the fix directly on the parameters / expected outcome Fernando originally reported in the issue.

Maybe not crucial, but probably not too much more work given that he provided the parameters and R output already.

Reply:
Added the tests in the last push